android/app/src/main/java/org/fossasia/openevent/data/SearchHeader.java

```diff
@@ -0,0 +1,22 @@
+package org.fossasia.openevent.data;
+
+/**
+ * Created by Shreyas on 5/15/2017.
+ */
+
+public class SearchHeader {
```

Comment:
Is there a need of such a container only holding a string? Rather simply use a string